src/shared/utils/thousands-separated-number.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2002-2019 "Neo4j,"
+ * Neo4j Sweden AB [http://neo4j.com]
+ * This file is part of Neo4j.
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+export default value =>
```

Comment:
Im going to have another look at this, but the general thought process is for defensive purposes but I dont know how defensive we need to be here
- `parseInt` to coerce everything to numbers, but need to do the `isNaN` check for null etc

It reads a bit odd anyway but open to suggestions, I will take another look in a bit